### DISCUSSION TITLE
Upcoming Cubit-Python webinar -- including an automated MOOSE study

### DISCUSSION THREAD

**Discussion by GregVernon** [idaholab/moose#17499] [Q&A Meshing] [answered]
Dear MOOSE community,

On April 6th at 10AM MDT I'll be giving a webinar on Coreform Cubit's Python API (which is also largely applicable to SNL's Cubit).  While it is a webinar, I intend for it to be instructive to new- and intermediate-users.  The webinar will be largely centered around live demonstrations of using the Python API.  One example that may be of particular interest to the MOOSE community is one that automates a MOOSE parameter-sweep study.  While not incredibly sophisticated, I hope that it might at least be insightful for those looking to automate their MOOSE simulation workflows.

If interested, [here's a link to register for the webinar.](https://us02web.zoom.us/webinar/register/WN_u_wF0nUDSIaIq0DUPIGrXA)

Hope to see you in the webinar!

Greg Vernon | Director of Product Management
Coreform LLC

**Comment by WilkAndy** [comment]
Hi @GregVernon , that's 2am for me.  Are you planning to upload to youtube/whatever,

a

**Reply by GregVernon** [reply]
@WilkAndy,

Yes!  We'll be recording the webinar and uploading to YouTube shortly after the webinar.  I'll reply with a YouTube link once we've done that.

**Reply by WilkAndy** [reply]
Thanks @GregVernon

**Reply by makeclean** [reply]
@GregVernon how did the webinar go?

**Reply by GregVernon** [reply]
I think it went mostly pretty well, we'll be uploading it within the next 24 hours or so to YouTube - and example scripts to Zenodo.

**Reply by WilkAndy** [reply]
Thanks @GregVernon

**Reply by GregVernon** [reply]
@WilkAndy - see my post below

**Comment by GregVernon** [comment, marked as answer]
Dear MOOSE Community,

Please see the recording of the webinar on YouTube: https://www.youtube.com/watch?v=2TCPPO8z5VA

I've also uploaded the example files (including the MOOSE study) to Zenodo: http://doi.org/10.5281/zenodo.4671051

I hope you found/find this webinar insightful, if not helpful!


Happy MOOSE-ing!

Greg
